__tests__/get.test.js

```diff
@@ -108,4 +108,33 @@ describe('/api/articles/:article_id', () => {
 
             })
     })
+})
+describe('/api/articles', () => {
+    test(`to GET status 200 and an array containing article objects containing the following properties:author, title, article_id, body, topic, created_at, votes, article_img_url, comment_count. the articles should be sorted by date in descending order.  `, () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then((response) => {
+                const articleArray = response.body.articles;
+                expect(articleArray.forEach((article) => {
```

Comment:
Make sure to assert a length when using a forEach, how many reviews are we expecting here?